projects/angular/src/emphasis/alert/_alert.clarity.scss

```diff
@@ -247,6 +265,13 @@
     }
 
     .alert-actions {
+      @include mixins.css-var(
+        display,
+        clr-link-display,
+        inline-block,
+        //needed for IE11
+        variables.$clr-use-custom-properties
+      );
```

Comment:
Do we need that and how it's connected with the `link` or the buttons?

Reply:
corrected in commit eb48c7f